packages/bundler/src/es6-module-bundler.ts

```diff
@@ -78,7 +78,7 @@ async function prepareBundleModule(
         getOrSetBundleModuleExportName(assignedBundle, sourceUrl, '*');
     bundleSource.body.push(babel.importDeclaration(
         [babel.importNamespaceSpecifier(babel.identifier(starExportName))],
-        babel.stringLiteral(rebasedSourceUrl)));
+        babel.stringLiteral(sourceUrl)));
```

Comment:
Looks like `rebasedSourceUrl` isn't used anymore in this function? Remove?

Reply:
Absolutely.  Good catch.  Removed.